shared/utils/rankEnum.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+export default Object.freeze({
```

Comment:
I'm not a fan of the `enum`, I think it's unnecessary when we're using types

Reply:
will use plain object 👍🏻